effekt/shared/src/main/scala/effekt/cps/PrettyPrinter.scala

```diff
@@ -150,8 +150,8 @@ object PrettyPrinter extends ParenPrettyPrinter {
     case Stmt.Resume(r, body, ks, k) =>
       "resume" <> parens(toDoc(r)) <+> toDoc(body) <+> "@" <+> toDoc(ks) <> "," <+> toDoc(k)
 
-    case Stmt.Hole() =>
-      "<>"
+    case Stmt.Hole(span) =>
+      "<>" <+> s"// @ ${span.range.from.format}"
```

Comment:
This is how I pretty-print the holes in IRs like Core or Machine in case you're interested